src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -166,6 +171,27 @@ class CardsResource extends Resource {
         created(cardResult).build()
     }
 
+    @PUT
+    @Path('{id}')
+    @Consumes(MediaType.APPLICATION_JSON)
+    Response putCard(@PathParam('id') IntParam id, @Valid ResultObject newResultObject) {
+
+        if(!cardDAO.cardExists(id.get())) {
+            return notFound().build()
+        }
+        Response badResponse = resultObjectValidator(newResultObject)
+        if(badResponse) {
+            return badResponse
+        }
+        cardDAO.putCard(id.get(), (Card)newResultObject.data.attributes)
+
+        Card card  = cardDAO.getCardById(id.get())
```

Comment:
These three lines at the end of this method are pretty similar to the ones at the end of the `POST` method. Maybe combine them into method that can be shared?